packages/odyssey-react-mui/src/TagList.tsx

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Stack, Tag } from ".";
+import { memo, ReactElement } from "react";
+
+export type TagListProps = {
+  children: ReactElement<typeof Tag> | Array<ReactElement<typeof Tag>>;
+};
+
+const TagList = ({ children }: TagListProps) => (
+  <Stack direction="row" spacing={2}>
```

Comment:
**Non-Blocking**

@KevinGhadyani-Okta this is another place we could use custom theming internally.

`direction` and `spacing` are style properties being handled here instead of `components.tsx` -- someone updating the styles there won't necessarily know they're being applied here. However, `Stack` is a generic component, so we wouldn't want to set the `defaultProps` on `MuiStack` just for this use case.

One solution would be to have a `variant` prop on `MuiStack`, similar to [what we do with MuiAlert](https://github.com/okta/odyssey/blob/develop/packages/odyssey-react-mui/src/theme/components.types.ts#L14). For ex: `variant="taglist"`. However, `MuiStack` has no `variant` prop to modify.

Reply:
Ah, so you're saying to wrap `Stack` to provide a cleaner API for Odyssey internally and externally?

Reply:
I agree 100%! We should be defining what these are.

We'll have to make a ticket for it.